packages/ckeditor5-table/src/ui/inserttableview.js

```diff
@@ -215,6 +203,36 @@ export default class InsertTableView extends View {
 		} );
 	}
 
+	/**
+	 * Creates a new Button for the grid.
+	 *
+	 * @private
+	 * @returns {module:ui/button/buttonview~ButtonView}
+	 */
+	_createGridButton( locale, row, column, label ) {
+		const button = new ButtonView( locale );
+		const bind = this.bindTemplate;
+
+		button.set( {
+			label,
+			withText: false,
+			'isOn': false
```

Comment:
`isOn` property is false by default too https://github.com/ckeditor/ckeditor5/blob/3b33b8fd8291b9bda184f1d3bdd94255e8aa12e3/packages/ckeditor5-ui/src/button/buttonview.js#L53